src/actions/navigation/index.ts

```diff
@@ -13,4 +15,16 @@ export const navigatorReset = (newScreen: NavigationNavigateActionPayload) => {
     index:0,
     actions: [navigate(newScreen)]
   })
-}
\ No newline at end of file
+}
+
+export const handleDeepLink = (url: string) => {
```

Comment:
Can we add a comment here perhaps? Something to help future maintainers quickly pick up what this does.